lib/audited/auditor.rb

```diff
@@ -302,6 +284,18 @@ def auditing_enabled
       def auditing_enabled=(val)
         Audited.store["#{table_name}_auditing_enabled"] = val
       end
+
+      protected
+      def default_ignored_attributes
+        [primary_key, inheritance_column] + Audited.ignored_attributes
+      end
+
+      def normalize_audited_options
+        audited_options[:on] = Array.wrap(audited_options[:on])
+        audited_options[:on] = [:create, :update, :destroy] if audited_options[:on].empty?
+        audited_options[:only] = Array.wrap(audited_options[:only]).map(&:to_s) if audited_options[:only]
```

Comment:
We want to keep `audited_options[:only]==nil` if it wasn't specified since we use it to determine which columns to save.